libs/mstable/flatcoin/src/components/TradingPanel/TradingRecap/TransactionOverview.tsx

```diff
@@ -1,28 +1,50 @@
 import { formatToUsd } from '@dhedge/core-ui-kit/utils';
+import { DEFAULT_MAX_SLIPPAGE } from '@frontend/shared-constants';
 import { TradingOverviewItem } from '@frontend/shared-ui';
 import { Stack } from '@mui/material';
+import { useIntl } from 'react-intl';
 
 import { useIsLeveragedType } from '../../../hooks/useIsLeveragedType';
 import { useTradingFees } from '../hooks/useTradingFees';
+import { useFlatcoinTradingState } from '../state';
 
 import type { StackProps } from '@mui/material';
 import type { FC } from 'react';
 
 const useTransactionOverview = () => {
   const isLeveraged = useIsLeveragedType();
   const tradingFees = useTradingFees();
+  const { slippage } = useFlatcoinTradingState();
 
-  return { tradingFees, isLeveraged };
+  return {
+    tradingFees,
+    isLeveraged,
+    slippage: slippage || DEFAULT_MAX_SLIPPAGE,
+  };
 };
 
 export const TransactionOverview: FC<StackProps> = (props) => {
-  const { tradingFees, isLeveraged } = useTransactionOverview();
+  const intl = useIntl();
+  const { tradingFees, isLeveraged, slippage } = useTransactionOverview();
   return (
     <Stack {...props} direction="column" spacing={1}>
       <TradingOverviewItem
         label="Fees"
         value={formatToUsd({ value: tradingFees })}
       />
+      {!isLeveraged && (
+        <TradingOverviewItem
+          label={intl.formatMessage({
+            defaultMessage: 'Max slippage',
+            id: 'k3YWIR',
+          })}
+          tooltipText={intl.formatMessage({
+            defaultMessage: 'We recommend 1%, but usually it will be < 1%.',
```

Comment:
may be we can use variables like ' We recommend {RECOMMENDED_SLIPPAGE}%, but usually it will be < {RECOMMENDED_SLIPPAGE}%. to make it more flexible.

Reply:
sounds good, will update it